examples/connect/src/app.tsx

```diff
@@ -0,0 +1,55 @@
+import {combineReducers, createStore} from 'redux';
+import {CheckBox, CheckBoxSelectEvent, Color, contentView, Stack} from 'tabris';
+import {DefaultRootState, injector, StateProvider} from 'tabris-decorators';
+import {register} from 'tabris-decorators';
+import {ExampleComponent} from './ExampleComponent';
+
+injector.jsxProcessor.unsafeBindings = 'error';
+
+declare module 'tabris-decorators' {
+
+  export interface DefaultRootState {
+    str: string;
+    num: number;
+  }
+
+}
+
+type RootState = DefaultRootState;
+
+type Actions = {
+  type: 'TOGGLE_VALUES',
+  checked: boolean
+};
+
+const store = createStore<RootState, Actions, unknown, unknown>(
+  combineReducers<RootState, Actions>({
+    num(state, action) {
+      if (action.type === 'TOGGLE_VALUES') {
+        return action.checked ? 90 : 10;
+      }
+      return state || 0;
```

Comment:
As a good practice, always use `??` to give defaults for unset values in TypeScript code? The state might be a valid falsy value.

Reply:
That is almost the same though? I don't see where `??` would behave more desireably than `||`.

Reply:
In the particular example it does not have any advantage. But it would be advantageous if you had a model where 0 was a valid value and you would provide a non-zero default.

`??` only handles unset values, so it has purer semantics without undesired side effects for this use case.